diagnostic.md

```diff
@@ -150,7 +193,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+All the divs but divOne and divFour will be floated to the left.
+divOne will be floated to the right and divFour will be stacked below to the left
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short:
```
2 3      1
4 5
```